cmd/otelcol/Dockerfile.windows

```
@@ -27,6 +27,23 @@ RUN Add-Type -AssemblyName System.IO.Compression.FileSystem ;\
     [System.IO.Compression.ZipFile]::ExtractToDirectory('agent-bundle_windows_amd64.zip', '.')
 RUN Remove-Item agent-bundle_windows_amd64.zip -force
 
+# Download JMX Metric Gatherer
+ARG JMX_METRIC_GATHERER_RELEASE
+RUN New-Item -Path "C:\\" -Name "opt" -ItemType "directory"
+RUN Invoke-WebRequest -Uri "https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v${env:JMX_METRIC_GATHERER_RELEASE}/opentelemetry-jmx-metrics.jar" -Outfile "c:\opt\opentelemetry-java-contrib-jmx-metrics.jar"
+
+ARG JDK_VERSION="11.0.19_7"
```

```suggestion
ARG JDK_VERSION="11.0.20_8"
```
Please rebase. The JDK was just updated to [`11.0.20_8`](https://github.com/signalfx/splunk-otel-collector/pull/3396) in the linux bundle.

---

Updated and rebased, thanks!